preferences/fedora/arm64/metadata/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: metadata
+  annotations:
+    tags: "hidden,kubevirt,fedora"
+    iconClass: "icon-fedora"
+    openshift.io/display-name: "Fedora (aarch64)"
```

Comment:
What's better to use as display name? aarch64 or arm64?

Reply:
`aarch64` is fine for the display name IMHO.